### PR TITLE
NodeJS: Use Copilot Chat and not copilot labs

### PR DESCRIPTION
**Pull request by tgrall**

### CODE REVIEW

**Comment by tgrall** [comment]
@microsoft-github-policy-service agree

**Review by cmendible — CHANGES_REQUESTED** [review]
Thanks for the update, please fix the typo and make sure all exercises are now based on the use of Copilot Chat.

File: .devcontainer/devcontainer.json
Comment (cmendible) [inline .devcontainer/devcontainer.json:1]: Please pull latest changes. The extension change was already merged.

**Review by cmendible — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json
Comment (tgrall) [reply .devcontainer/devcontainer.json:1]: Done

**Comment by tgrall** [comment]
> Thanks for the update, please fix the typo and make sure all exercises are now based on the use of Copilot Chat.

@cmendible : 
- I have added back the labs that are using the `sample.txt` that was missing 
- I fix the typo on `language` at the beginning, let me know if you have found more.

Note: my teammate @Msanzdelrio  is working on the dotnet labs

**Comment by Msanzdelrio** [comment]
> > Thanks for the update, please fix the typo and make sure all exercises are now based on the use of Copilot Chat.
> 
> @cmendible :
> 
> * I have added back the labs that are using the `sample.txt` that was missing
> * I fix the typo on `language` at the beginning, let me know if you have found more.
> 
> Note: my teammate @Msanzdelrio is working on the dotnet labs

I have already submitted my PR and has already been merged :)

**Review by cmendible — APPROVED** [review]
LGTM

**Comment by cmendible** [comment]
@Msanzdelrio @tgrall Thanks for contributing!

**Pull request merged**